.github/workflows/dev-release.yml

```diff
@@ -18,3 +19,35 @@ jobs:
       - run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN_DEV }}" > .npmrc
       - run: ./scripts/dev-version.sh
       - run: yarn publish
+
+  changeset-pr:
+    name: Create PR for changeset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3.0.0
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+          # This allows the changesets/action step to override the default actions user,
+          # and allow CI jobs on commits coming from that step.
+          # https://github.com/changesets/action/issues/70
+          persist-credentials: false
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
+      - run: npm i -g yarn@1.22.17
+      - run: yarn --frozen-lockfile
+      - run: yarn build
```

Comment:
We don't actually need to build it (although it doesn't hurt). Consider removing it for speed.